examples/async.rs

```diff
@@ -0,0 +1,81 @@
+ethcontract::contract!("examples/truffle/build/contracts/RustCoin.json");
+
+#[rustversion::since(1.39)]
+fn main() {
+    use ethcontract::transaction::Account;
+    use ethsign::SecretKey;
+    use futures::compat::Future01CompatExt;
+    use web3::api::Web3;
+    use web3::transports::Http;
+    use web3::types::{Address, H256};
+
+    async fn print_balance_of(instance: &RustCoin, account: Address) {
+        let balance = instance
+            .balance_of(account)
+            .execute()
+            .await
+            .expect("balance of");
+        println!("Account {:?} has balance of {}", account, balance);
```

Comment:
Are we expecting to keep these print statements or are these really meant to do as the function says?

Reply:
This is just part of the example, just you see something happen when you run it. This is not actually part of the library.